internal/controller/server/client_services.go

```diff
@@ -84,9 +84,9 @@ func (s *clientCPServicesServer) GetStreamLogs(request *clientCPproto.RequestFor
 	return nil
 }
 
-//ExecuteJob will call Executejob function of execution and get jobId
-func (s *clientCPServicesServer) ExecuteJob(ctx context.Context, execute *clientCPproto.RequestForExecute) (*clientCPproto.Response, error) {
-	jobId, err := s.procExec.ExecuteJob(ctx, execute.JobName, execute.JobData)
+// ExecuteJob will call ExecuteJob function of execution and get jobId
+func (s *clientCPServicesServer) ExecuteJob(ctx context.Context, jobContext *clientCPproto.RequestForExecute) (*clientCPproto.Response, error) {
```

Comment:
Dont use the name jobContext for arguments we are sending in for executing a job. Use a better name which conveys the meaning simply